RELEASE.md

```diff
@@ -1,5 +1,12 @@
 # Upcoming Release
 
+## Major features and improvements
+## Bug fixes and other changes
+## Breaking changes to the API
+## Documentation changes
+
+# Release 0.19.11
+
 ## Major features and improvements
```

Comment:
Shall we maybe remove the "## Breaking changes to the API" and "## Documentation changes" if they're empty?